### ISSUE TITLE
Feature: PDF Image Annotation

### ISSUE THREAD

**Issue by seveibar** [UniversalDataTool/universal-data-tool#49]
Support PDFs in Image Segmentation and Image Classification. Please thumbs up if you want it.

**Comment by Ownmarc** [comment]
Been working with PDF in the past and, IMO, the best is to convert everything to JPG using a lib like pdf2image or something similar.

If we want to make something that is not well supported by other annotation tools or lib, something like PDF2text with a 2D mapping between the raw text and the original PDF would be insane. This could then be used for NLP tasks or vision tasks to find the right zones to get the information needed. 

This is not easy to do, in Python I use pdfminer and pypdf2 to extract text. pdfminer can return coords of each letter/words while pypdf can't. Simple pdf decryption like password are supported, but no support for online request to decrypt (that require things like FileOpen)

**Comment by seveibar** [comment]
Agreed that combining the NER/NLP text tasks with PDFs would be an amazing feature.

As far as PDF viewing goes, there are two solutions I think will be pretty good
1. Desktop Application transform that converts PDF to Image. Then use image segmentation normally.
2. Support PDFs in the underlying react-image-annotate library by using an in-browser PDF renderer. 

Approach (2) would work on web and would provide a nicer end user experience. Approach (1) is a bit easier. You could also implement both of these because some may prefer (1) for building their model anyway.
